src/slider.tsx

```diff
@@ -0,0 +1,269 @@
+import * as React from 'react';
+import {useScreenSize, useTheme} from './hooks';
+import * as styles from './slider.css';
+import classnames from 'classnames';
+import IntegerField from './integer-field';
+import Tooltip from './tooltip';
+
+interface SliderProps {
+    disabled?: boolean;
+    steps?: number | Array<number>;
+    max?: number;
+    min?: number;
+    field?: boolean;
+    value?: number;
+    onChange?: (value: number) => void;
+    getStepArrayIndex?: (value: number) => void;
+    invalidText?: string;
```

Comment:
this prop doesn't apply, right? It is covered with helperText

Reply:
Yes, however it needs to be present to have the default message of 'Max:' slider limit, so I will convert it to a const instead of a prop